apps/storybook/.storybook/preview.tsx

```diff
@@ -150,9 +150,22 @@ const preview: Preview = {
         dynamicTitle: true,
       },
     },
+    colorScheme: {
+      description: 'Set color-scheme in stories',
+      defaultValue: 'light',
+      toolbar: {
+        title: 'Theme',
+        items: [
+          { title: '☀️ Light', value: 'light' },
+          { title: '🌙 Dark', value: 'dark' },
```

Comment:
```suggestion
          { title: 'Light', value: 'light' },
          { title: 'Dark', value: 'dark' },
```
Just a suggestion, @mimarz can decide here.
I am not a fan of using emojis 😅 They look different from system to system

Reply:
Removed he emojis